### ISSUE TITLE
cannot import name 'SlashCommandOptionType' from 'discord.enums'

### ISSUE THREAD

**Issue by brywiaon** [CoccodrillooXDS/TTS-bot#3]
The code attempts to import Option from discord.commands. However, this error message pops up. It appears the code uses both discord.py and pycord, which seem to clash with each other as uninstalling pycord leads to an error saying that discord.abc was not found. How do I resolve this?

**Comment by CoccodrillooXDS** [comment]
Hi!
This bot uses the **py-cord** library. If you have both libraries, they will go in conflict with each other as they use the same library name ("**discord**").

Make sure to uninstall discord.py or create a [venv](https://docs.python.org/3/library/venv.html) with the requirements from "requirements.txt" before running the bot.

**Comment by brywiaon** [comment]
Hi!

I have uninstalled discord.py and updated pycord to the latest verison with pip.
However, I am currently getting an error message on the same import line above, which says "No module named discord.abc".
How do I solve this?

**Comment by CoccodrillooXDS** [comment]
Hi!
I'm unable to reproduce your issue, as my code doesn't import anything from "discord.abc". Make sure you are using the correct libraries specified in [requirements.txt](https://github.com/CoccodrillooXDS/TTS-bot/blob/master/requirements.txt).

You can install them using 
> pip install -r requirements.txt

or with

> python -m pip install -r requirements.txt

**Comment by brywiaon** [comment]
I got it to work by reinstalling pycord. However, there was another issue caused by line 227 in file bot.py where "source" should be "source=source". I discovered this after running into this error:

AttributeError: 'FFmpegPCMAudio' object has no attribute '_process'
FFmpegPCMAudio.__init__() missing 1 required positional argument: 'source'

 After changing this, the bot seems to be working perfectly now. Thank you!
